pkg/oidcagent/pkg/cookie/README.md

```diff
@@ -0,0 +1,105 @@
+# Cookie store for [Session](https://github.com/go-session/session)
```

Comment:
Looks like you imported this twice? once as `pkg/cookie` and once as `pkg/oidcagent/pkg/cookie` ?

Reply:
oops. fixed.